packages/eslint-config-base/package.json

```diff
@@ -26,7 +26,7 @@
     "@stylistic/eslint-plugin-js": "2.8.0",
     "confusing-browser-globals": "1.0.11",
     "eslint-import-resolver-typescript": "3.6.1",
-    "eslint-plugin-import": "npm:eslint-plugin-i@2.29.1",
+    "eslint-plugin-import-x": "4.3.0",
```

Comment:
I see that [v4.3.1](https://github.com/un-ts/eslint-plugin-import-x/releases/tag/v4.3.1) has been updated.

Reply:
Updated to use the latest `eslint-plugin-import-x@4.3.1`